README.md

```diff
@@ -144,20 +158,19 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
     ```
     * Exit psql by typing in \q and hitting enter.
 * Migrate
-    ```
-
+    ```python
     python manage.py makemigrations caffe_app
     python manage.py migrate
     ```
 8. Install node modules
-```
+```shell
 npm install
 npm install --save-dev json-loader
 sudo npm install -g webpack
 webpack --progress --watch --colors
 ```
 
-9. To start celery worker
+9. Start celery worker
 ```
```

Comment:
Can you mention here that you need celery worker needs to be executed parallely in a separate terminal